src/algorithms/calorimetry/ImagingTopoClusterConfig.h

```diff
@@ -29,4 +35,31 @@ namespace eicrecon {
 
   };
 
+  std::istream& operator>>(std::istream& in, ImagingTopoClusterConfig::ELayerMode& layerMode) {
+    std::string s;
+    in >> s;
+    // stringifying the enums causes them to be converted to integers before conversion to strings
+    if (s == "etaphi" or s=="0") {
+        layerMode = ImagingTopoClusterConfig::ELayerMode::etaphi;
+    } else if (s == "xy" or s=="1") {
```

Comment:
Let's mark this as a future issue. We should not have to specify as "0" or "1", but let's not let it hold up this PR.

Reply:
I.e. please file this as an issue :-)

Reply:
ok. I will file this as an issue.